docs/using/examples/context.rst

```diff
@@ -54,9 +54,13 @@ variables.
            :caption: :fab:`java` Env.java
```

Comment:
as a meta comment, I don't think `Context` is a very descriptive name for this docs page